examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,63 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method calculates different quantities
+inside blocks/bins whose dimensions are defined via the ``spacing`` parameter.
+The following examples show how to calculate the averages of the given values inside
+each block and how to report the number of points inside each bin.
```

Comment:
```suggestion
The following examples show how to calculate the averages of the given values 
inside each block and how to report the number of points inside each bin.
```
This should fix the line length issue in the style check